### ISSUE TITLE
Saved loops click at loop point

### ISSUE THREAD

**Issue by levicole** [schollz/oooooo#24]
Hopefully this makes sense. I can upload a session here if that's helpful.  After I save a session, and then load it again, the loops have a click once they've cycled all the way through back to the beginning.

I haven't played around too much, but the times I've saved sessions that I've loved, they've usually been guitar heavy, and so maybe if there was some silence at the loop point, the click might not be there?

I updated to the latest version today in hopes that the cross fade would help, but it seems to still be there.

**Comment by schollz** [comment]
that absolutely makes sense and I know the reason. I'm only saving the length of the loop, but I actually need to save the loop plus a little bit extra (which provides the fade to remove clicking). will fix!

**Comment by schollz** [comment]
fixed!

**Comment by schollz** [comment]
(you'll have to re-record your things unfortunately)

**Comment by levicole** [comment]
Not a big deal! I like the feature of saving, but I do like to think of most things I do on norns as ephemeral. Makes each time I work with it a little gift to myself.

What I have done in the past is just shorten the loop length by a milisecond or so, and that's gotten rid of the click.

I experienced this when I first started using oooooo, then I moved and didn't use much of my gear for a couple of month, and this morning's session re-jogged my memory.

I appreciate all that you do for the Norns community. I love the work you do, and enjoyed your SC classes!
